core-bundle/src/EventListener/PrettyErrorScreenListener.php

```diff
@@ -107,7 +107,9 @@ private function handleException(ExceptionEvent $event): void
                 break;
 
             case $exception instanceof ServiceUnavailableHttpException:
-                $this->renderTemplate('service_unavailable', 503, $event);
+                if (!$this->renderErrorScreenByType(503, $event)) {
+                    $this->renderTemplate('service_unavailable', 503, $event);
+                }
```

Comment:
```suggestion
                $this->renderErrorScreenByType(503, $event);

                if (!$event->hasResponse()) {
                    $this->renderTemplate('service_unavailable', 503, $event);
                }
```

This way we do not have to add a return value to the `renderErrorScreenByType()` method.